compiler/fir/fir2ir/testData/ir/irText/properties/backingField/independentBackingFieldType.ir.txt

```diff
@@ -48,7 +48,7 @@ FILE fqName:<root> fileName:/independentBackingFieldType.kt
       VAR name:a type:<root>.A [val]
         CONSTRUCTOR_CALL 'public constructor <init> () [primary] declared in <root>.A' type=<root>.A origin=null
       VAR name:it type:kotlin.Int [val]
-        CALL 'public final fun and (other: kotlin.Int): kotlin.Int [operator] declared in kotlin.Int' type=kotlin.Int origin=null
+        CALL 'public final fun and (other: kotlin.Int): kotlin.Int [infix] declared in kotlin.Int' type=kotlin.Int origin=null
```

Comment:
I have some doubts about this change. This test file is marked as `FIR_IDENTICAL`, so probably it will break PSI2IR tests. Should we change PSI2IR in the same way?